GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloCMBlockIII+.cfg

```diff
@@ -620,3 +632,11 @@ PART
 		RFBand = S		//Default to S-band
 	}
 }
+
+//	================================================================================
+//	Override RO global settings to get our temperatures back
+//	================================================================================
+@PART[ROC-ApolloCMBlockIII]:FOR[RealismOverhaul]
```

Comment:
This needs to be AFTER[RealismOverhaul] since this is a separate mod.
Functionally, ROCapsules being lexicographically after RealismOverhaul means that anything we put in "FOR:[RealismOverhaul]" will run after every FOR[RealismOverhaul] patch located in the RealismOverhaul folder, anyway.
This comment applies throughout the entire PR.

Reply:
Patching syntax corrected.